packages/kit/src/runtime/client/client.js

```diff
@@ -1,5 +1,5 @@
 import { BROWSER, DEV } from 'esm-env';
-import { onMount, tick } from 'svelte';
+import * as svelte from 'svelte';
```

Comment:
why change the import style here?

Reply:
Because `unstate` isn't available in Svelte 4, so it would instantly break